h/static/styles/partials-v2/_search.scss

```diff
@@ -0,0 +1,91 @@
+// Search page
+// -----------
+// Styles for the search page which displays the results
+// of activity queries.
+
+.search {
```

Comment:
CSS component names should generally be nouns, so `search-result-container` rather than `search` for example.